backend/clubs/migrations/0035_clubvisit.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.1 on 2020-08-25 19:13
+
+from django.conf import settings
```

Comment:
Remember to merge the migrations before merging this branch, Git won't merge them for you.